config/stored_requests.go

```diff
@@ -71,6 +71,8 @@ type StoredRequests struct {
 	// HTTPEvents configures an instance of stored_requests/events/http/http.go.
 	// If non-nil, the server will use those endpoints to populate and update the cache.
 	HTTPEvents HTTPEventsConfig `mapstructure:"http_events"`
+	// Timeout defines the number of milliseconds before a timeout occurs with stored requests fetch
+	Timeout int `mapstructure:"timeout_ms"`
```

Comment:
The point was raised in one of the closed related issues (https://github.com/prebid/prebid-server/pull/1533#discussion_r503389380) that perhaps this should be a top level config. Moving it down a level as you have does increase flexibility though as it allows for different timeouts for each data type that may be fetched (requests, responses, accounts, categories, etc). Let's discuss with the group.

Reply:
Sounds good, we can bring this up tomorrow!

Reply:
Discussed Today: I'm investigating the tradeoff between leaving the timeout config at this level and updating the other various config variable usages that are also of type `StoredRequests` to utilize the timeout appropriately, or if it would be easier to just have this variable at the root level.